docs/ml-classification-regression.md

```diff
@@ -302,6 +302,38 @@ Refer to the [Java API docs](api/java/org/apache/spark/ml/classification/OneVsRe
 </div>
 </div>
 
+## Naive Bayes
+
+[Naive Bayes](http://en.wikipedia.org/wiki/Naive_Bayes_classifier) are a family of simple 
+probabilistic classifiers based on applying Bayes' theorem with strong (naive) independence 
+assumptions between the features. More information about the spark.ml implementation can be 
+found further in the section on [Naive Bayes in MLlib](mllib-naive-bayes.html#naive-bayes-sparkmllib).
+
```

Comment:
I think it's better to clarify ml.NaiveBayes supports Multinomial NB and Bernoulli NB. Meanwhile, we should provide the link to corresponding documents. You can refer [the NaiveBayes API doc](https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/classification/NaiveBayes.scala#L67).